wpeverest/everest-forms
# Enhancement - Select All or Unselect All option in Checkboxes and Dropdown(multi-select) field

### All Submissions:



### Changes proposed in this Pull Request:
Previously, there was no 'Select All' option in 'Checkboxes' field and 'Select All' & 'Unselect All' options in 'Dropdown' field. This PR provides these options.

### How to test the changes in this Pull Request:

1. Create a form with containing at least one  'Checkboxes' field.
2. Go to 'Advanced Options' of that 'Checkboxes' field and enable 'Select All'. And save this form.
3. Preview this form and verify 'Select All' option works properly or not.

### Types of changes:



### Other information:



### Changelog entry

> Enhancement - Select All or Unselect All option in Checkboxes and Dropdown(multi-select) field.
